src/lib.rs

```diff
@@ -298,7 +298,9 @@
 //! #[macro_use(await)]
 //! extern crate nb;
 //!
-//! use std::ops::Generator;
+//! extern crate core;
```

Comment:
```suggestion
```

Reply:
It does not compile otherwise.

Reply:
Superweird. Probably needs a thourough overhaul.